pkg/bisect/bisect.go

```diff
@@ -362,11 +360,14 @@ func (env *env) minimizeConfig() (*testResult, error) {
 		if err != nil {
 			return 0, err
 		}
-		testResults[hash.Hash(test)] = testRes
+		if testRes.verdict == vcs.BisectBad {
+			// Only remember crashes.
```

Comment:
why?

Reply:
I think what we really want here is to get the new crash report, after minimization. If we don't do the `testRes.verdict == vcs.BisectBad` check, we will likely just return an empty one.